test/e2e/atlas/setup_failure_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build e2e || (atlas && interactive)
+
+package atlas_test
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongodb-atlas-cli/test/e2e"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSetupFailureFlow(t *testing.T) {
+	g := newAtlasE2ETestGenerator(t)
+	g.generateProject("setup")
+	cliPath, err := e2e.AtlasCLIBin()
+	req := require.New(t)
+	req.NoError(err)
+
+	t.Run("Invalid Public Key", func(t *testing.T) {
+		t.Setenv("MCLI_PUBLIC_API_KEY", "invalid_public_key")
+		cmd := exec.Command(cliPath,
+			"setup",
+			"--skipMongosh",
+			"--skipSampleData",
+			"--force")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		req.Error(err)
+		assert.Contains(t, string(resp), "Unauthorized", "Expected unauthorized error due to invalid public key.")
+	})
+
+	t.Run("Invalid Private Key", func(t *testing.T) {
+		t.Setenv("MCLI_PRIVATE_API_KEY", "invalid_private_key")
+		cmd := exec.Command(cliPath,
+			"setup",
```

Comment:
Can you add this as a constant in [here](https://github.com/mongodb/mongodb-atlas-cli/blob/09a1900f0c02caa2df2a8321bec4eb413393786a/test/e2e/atlas/helper_test.go#L37)?